kernel/src/engine/default/json.rs

```diff
@@ -475,15 +591,13 @@ mod tests {
         ];
         let paths = paths.map(|p| std::fs::canonicalize(PathBuf::from(p)).unwrap());
 
-        let path_string = paths[0].to_string_lossy().to_string();
-        let object_store_path = Path::from(path_string);
-        // for the first 000000.json, wait for 1 second
-        let key_map = (object_store_path, Duration::from_secs(1));
-
-        let store = Arc::new(SlowGetStore::new(
-            LocalFileSystem::new(),
-            vec![key_map].into_iter().collect(),
-        ));
+        // object store will resolve GETs to paths in reverse: 0001.json, 0000.json
+        let reverse_paths = paths
```

Comment:
avoid turbofish:
```suggestion
        let reverse_paths: Vec<_> = paths
```
Also, `OrderedGetStore::new` takes `impl Into<VecDeque>`, so we can just pass a `Vec` (which has zero-cost conversion to `VecDeque`)